### PR TITLE
fix #60131 filmativa.xyz

### LINKED ISSUE #60131: filmativa.xyz
### Issue URL (Incorrect Blocking)
 [https://filmativa.xyz/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ffilmativa.xyz%2F)
### Comment
Username: @ernestsehic
> Vec sam jednom prijavio da nema titlova, i to je neko uradio azuriranje i rijesio se problem. Sad opet web stranica ima problem sa prikazom titlova kad radi adblock...
> --
> Podaci u nastavku automatski se generiraju. Nemojte ih brisati.
> 
> Referrer:
> 
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/adguard-wot/1.0/wot.meta.js
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.meta.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.meta.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Mac v2.5.0.838 nightly
Browser: | Safari
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (180), <br/>Block Push API, <br/>Block Location API, <br/>Block Java
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard German
Custom filter subscriptions: |

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#60131

### CODE REVIEW

File: EnglishFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -230,6 +230,9 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/60124
 !+ PLATFORM(ext_ff)
 @@||myhdjav.net^$content
+! https://github.com/AdguardTeam/AdguardFilters/issues/60131
+@@||doodstream.com^$font,image,script,stylesheet,xmlhttprequest,other,domain=dood.to|dood.watch
+@@||dood.watch^$subdocument
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/whitelist.txt:235]: Iframe with video doesn't load f.e. in Chrome.

Comment (Alex-302) [reply EnglishFilter/sections/whitelist.txt:235]: I don't see doodstream.com requests.
@zloyden

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/whitelist.txt:235]: > Iframe with video doesn't load for example in Chrome.

Looks like they have changed something, now I don't see this request too and subs works good with existing rules

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/whitelist.txt:234]: And now I see this request again
Screenshot 2020-07-30 at 14 48 46

**Review by Alex-302 — APPROVED** [review]

File: EnglishFilter/sections/whitelist.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/whitelist.txt:234]: I have the same
<details><summary>Screenshot:</summary>

image

</details><br/>

**Pull request merged**